scripts/cnode-helper-scripts/env

```diff
@@ -954,6 +955,8 @@ set_default_vars() {
   [[ -z ${MITHRIL_HOME} ]] && MITHRIL_HOME="${CNODE_HOME}/mithril"
   [[ -z ${MITHRIL_SIGNER_ENABLED} ]] && MITHRIL_SIGNER_ENABLED="N"
   [[ -z ${STRICT_VERSION_CHECK} ]] && STRICT_VERSION_CHECK="Y"
+  [[ -z "${KOIOS_API_HEADERS[*]}" ]] && KOIOS_API_HEADERS=()
+  [[ -n "${KOIOS_API_TOKEN}" ]] && KOIOS_API_HEADERS+=(-H "'Authorization: Bearer ${KOIOS_API_TOKEN}'")
```

Comment:
I suspect that env is being sourced/called multiple times while the logic used presumed a single call to **set_default_vars()**. This would allow the token to be inserted more than once.

```suggestion
  if [[ -z "${KOIOS_API_HEADERS[*]}" ]] ; then
    if [[ -n "${KOIOS_API_TOKEN}" ]] ; then
      KOIOS_API_HEADERS=(-H "'Authorization: Bearer ${KOIOS_API_TOKEN}'")
    else
      KOIOS_API_HEADERS=()
    fi
  fi
```